.circleci/config.yml

```diff
@@ -16,8 +16,8 @@ jobs:
         type: string
         default: ""
     docker:
-      - image: "circleci/golang:<< parameters.version >>"
-    working_directory: /go/src/github.com/gorilla/websocket
+      - image: "cimg/go:<< parameters.version >>"
+    working_directory: /home/circleci/project/go/src/github.com/gorilla/websocket
```

Comment:
https://circleci.com/docs/2.0/env-vars#using-parameters-and-bash-environment seems like hardcoding is necessary (i.e. can't use `$HOME`, it seems)